Dockerfile

```diff
@@ -1,17 +1,20 @@
+#BASE_URI_ARG argument is passed to dockerfile from command line to rebase the ui path For eg: BASE_URI_AR=/my-ui/. 
+#If passed empty, then / is appended. The Uri must begin and end with /  
 FROM node:alpine as build-deps
+ARG BASE_URI_ARG
 WORKDIR /usr/src/app
 COPY package.json package-lock.json ./
 RUN npm install
 COPY . ./
-RUN npm run build -- --configuration production --aot
+RUN npm run build -- --configuration production --base-href=${BASE_URI_ARG:-/} --aot
```

Comment:
I feel this is not what we agreed.  This locks the artifact to that specific path.

Note this is a container build phase you are modifying.

Reply:
True.. this is in the build phase. I had to recheck with Martin. we could add the suffix to ui instance, would solve the problem of the routing. The existing ui refers directly to the main route /. 

for an eg: below routing in ingress  with existing ui. 

        - pathType: Prefix
          path: /
          backend:
            service:
              name: dcsa-edoc-ui
              port:
                number: 80

Reply:
My understanding was still that we agreed with Alan that we have the same artifact for all environments, which sounds like we should **not** replace one hard code the base ref in the compiled artifact with another hard coded base ref.  However, this is exactly with this PR is doing, so either something changed without my knowing of it or this PR is not doing what we agreed.